### ISSUE TITLE
403 Error - Solution does not work anymore

### ISSUE THREAD

**Issue by JonasBirk** [kevmo314/faster-twitter#1]
I was so happy that I found this solution but it does not seem to work anymore. Probably since all this twitter api changing stuff. 

Best, 
Jonas

**Comment by kevmo314** [comment]
Yeah, unfortunately with the twitter API locked down this is no longer possible :(
